src/readers/reader_tum.py

```diff
@@ -0,0 +1,101 @@
+import open3d as o3d
+import mrob
+import numpy as np
+import os
+import cv2
+
+from reader import Reader
+from nptyping import NDArray, Shape, Float, UInt8
+from pathlib import Path
+from src.database import Database
+
+
+class ReaderTUM(Reader):
+    def __init__(self, path_to_dataset: str, max_difference: int = 1000):
+        self.path_to_dataset = path_to_dataset
+        self.max_difference = max_difference
+
+    def read_dataset(self) -> Database:
+        return super().read_dataset()
```

Comment:
why is override required?

Reply:
Because we must implement all parent methods? Maybe I don't understand your question or don't know smth about Python :)